stablehlo/reference/Ops.cpp

```diff
@@ -328,6 +328,25 @@ SmallVector<InterpreterValue> eval(Region &region,
       auto operand = scope.findTensor(clzOp.getOperand());
       auto result = evalClzOp(operand, clzOp.getType());
       scope.add(clzOp.getResult(), result);
+    } else if (auto collectiveBroadcastOp =
+                   dyn_cast<CollectiveBroadcastOp>(op)) {
+      auto operand = scope.findTensor(collectiveBroadcastOp.getOperand());
+
+      auto replicaGroupsAttr = collectiveBroadcastOp.getReplicaGroups();
+      auto replicaGroupsShape = replicaGroupsAttr.getShapedType().getShape();
+      SmallVector<SmallVector<uint32_t>> replicaGroups(replicaGroupsShape[0]);
+      auto replicaGroupsIt = replicaGroupsAttr.getValues<int64_t>().begin();
+      for (auto &replicaGroup : replicaGroups)
+        for (auto i = 0; i < replicaGroupsShape[1]; ++i, ++replicaGroupsIt)
+          replicaGroup.push_back(*replicaGroupsIt);
```

Comment:
Would a call to `getReplicaGroups` work here? I'm noticing that `AllToAll` and `AllGather` also have this code.

Reply:
`getReplicaGroups` returns a `DenseIntElementsAttr`, but we still need to convert it to a list of lists to pass it to the eval function.

Reply:
Ah good point. I notice that this code is duplicated in a few places though, consider adding a helper.